compiler/internal/codegen/codegen_main.go

```diff
@@ -131,7 +131,7 @@ func (b *Builder) Main() (f *File, err error) {
 			subscriptions := Dict{}
 
 			for _, sub := range topic.Subscribers {
-				traceID := int(b.res.Nodes[sub.DeclFile.Pkg][sub.CallSite].Id)
+				traceID := int(b.res.Nodes[sub.DeclFile.Pkg][sub.IdentAST].Id)
```

Comment:
Due to now treating subscriptions as a resource, rather than resource usage - this changed what node we register the reference from the call site to the ident that the subscriber is bound to.